asmcomp/cmmgen.ml

```diff
@@ -3084,6 +3085,7 @@ let compunit (ulam, preallocated_blocks, constants) =
   let c1 = [Cfunction {fun_name = Compilenv.make_symbol (Some "entry");
                        fun_args = [];
                        fun_body = init_code; fun_fast = false;
+                       fun_fast_compile = true;
```

Comment:
You should link to the relevant MPR to justify the specific choice of enabling `fast_compile` for compilation-unit functions.